tests/test_ren_pool.py

```diff
@@ -102,3 +102,27 @@ def test_withdraw_fullfilment(owner, ren_pool, ren_token, user, amount):
     assert ren_pool.balanceOf(user) == amount
     assert ren_pool.isLocked() == True
     assert ren_pool.totalPooled() == C.POOL_BOND
+    
+@given(
+    user=strategy('address'),
+    amount=strategy('uint256', min_value = 1, max_value = C.POOL_BOND)
+)
+def test_cancel_withdraw_request(owner, ren_pool, ren_token, user, amount):
+    """
+    Test Withdraw Cancellation
+    """
+    assert ren_pool.totalPooled() == 0
+    assert ren_pool.isLocked() == False
+    
+    ren_token.approve(ren_pool, C.POOL_BOND, {'from': owner})
+    ren_pool.deposit(C.POOL_BOND, {'from': owner})
+    assert ren_pool.isLocked() == True
+    assert ren_pool.totalPooled() == C.POOL_BOND
+    
+    # Requesting withdraw
+    ren_pool.requestWithdraw(amount, {'from': owner})
+    assert ren_pool.withdrawRequests(owner) == amount
```

Comment:
here we could use the `user` variable instead of the `owner` so that the test is more generic (maybe this works for the owner but not for the admin or some other regular user?). On line 107 we define the `user` variable which is a generic address (could be any address from the accounts array generated by ganache). Have a look at the brownie docs to see how test strategies can be used.